tests/accumulator_test.py

```diff
@@ -263,17 +264,41 @@ class SumAccumulatorTest(unittest.TestCase):
 
     def test_without_noise(self):
         sum_accumulator = accumulator.SumAccumulator(
-            accumulator.SumParams(), list(range(6)))
+            accumulator.SumParams(
+                MeanVarParams(1, 1, 0, 0, 1, 1,
+                              pipeline_dp.NoiseKind.GAUSSIAN)), list(range(6)))
         self.assertEqual(sum_accumulator.compute_metrics(), 15)
 
         sum_accumulator.add_value(5)
         self.assertEqual(sum_accumulator.compute_metrics(), 20)
 
         sum_accumulator_2 = accumulator.SumAccumulator(
-            accumulator.SumParams(), list(range(3)))
+            accumulator.SumParams(
+                MeanVarParams(1, 1, 0, 0, 1, 1,
```

Comment:
This test (w/o noise) makes sense. It's possible to achieve that the result is almost w/o noise, by setting very high (eps, delta) - like eps = 1e20, delta = 1 (delta might be considered as probability, it's in range [0, 1])